File: OpenLabHub/OpenLabHub/OpenLabHub/Models/Application.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpenLabHub.Models
{
    public class Application
    {
        public string state { get; set; }
        public string message { get; set; }
        public User user { get; set; }
        public Project project { get; set; }
    }
}

File: API/SpaceAppChallenge/src/main/java/com/example/spaceappchallenge/model/Application.java
package com.example.spaceappchallenge.model;

import com.example.spaceappchallenge.enums.State;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table(name="application")
public class Application {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long applicationId;
    private State state;
    private String message;
    @ManyToOne(cascade = CascadeType.PERSIST)
    @JoinColumn(name="user_id", nullable = false)
    private User user;
    @ManyToOne(cascade = CascadeType.PERSIST)
    @JoinColumn(name="project_id", nullable = false)
    private Project project;
}
